util.c

```diff
@@ -201,3 +201,57 @@ int r_mkdir(char *path)
 	}
 	return 0;
 }
+
+spawn_t spawn(const char *cmd, char *const argv[], unsigned int flags)
```

Comment:
The function is currently not very efficient if the caller is interested in only read or write.

Since this will be called rather often (via `open_info()` and `win_title_get()` in #213), I think it makes sense to optimize the function to avoid doing unnecessary work.

Reply:
After profiling bit, doesn't seem to be necessary.